_config.yml

```diff
@@ -40,3 +42,4 @@ plugins:
 - jekyll-gist
 - jekyll-feed
 - jekyll-include-cache
+- jekyll-multiple-languages-plugin
```

Comment:
What does this plugin do for us?

Reply:
Handles the page redirects (/en/, /es/, etc.). Gives a few tags to make setting up language stuff on our Jekyll pages easily.